.github/workflows/playwright.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     needs: [build-plugin]
     strategy:
       matrix:
-        testSuite: ['elements-regression', 'default', 'nested-tabs', 'reverse-columns']
+        testSuite: ['elements-regression', 'default', 'nested-tabs', 'reverse-columns', 'containerGrid']
```

Comment:
Should we maintain the convention we are currently using?
I think it should be `container-grid`